### PR TITLE
Adds an Economy Manager to the Admin Event Panel

### PR DESCRIPTION
**Pull request by Sirryan2002**
## What Does This PR Do
Adds an event panel for Admins to mess with the economy, gives them the tools to modify payrolls and change when the next payroll is coming. If they want to specifically interact with a money account and suspense/check transactions, the account database terminal exists for that and is easy to use.

## Why It's Good For The Game
muh admin tooling

## Images of changes
economyadminpanel

## Testing
I pressed all the buttons and it did stuff

## Changelog
:cl:
add: Admin Event panel for the economy
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
more verbs :death:

File: code/modules/economy/economy_admin.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		if("accelerate_payroll")
+			var/num_input = input(ui.user, "Amount", "Accelerate payroll by how many minutes?") as null|num
+			if(isnull(num_input))
+				return
+			SSeconomy.next_paycheck_delay -= num_input * (60 SECONDS)
+		if("set_payroll")
+			var/num_input = input(ui.user, "Amount", "Set payroll delay to how many minutes?") as null|num
+			if(isnull(num_input))
+				return
+			SSeconomy.next_paycheck_delay = world.time + (num_input * (60 SECONDS))
```
Comment (farie82) [inline code/modules/economy/economy_admin.dm:90]:
```suggestion
			SSeconomy.next_paycheck_delay = world.time + (num_input * 60 SECONDS)
```
Can even remove the other brackets. Same for the other instances above

File: code/modules/economy/economy_admin.dm [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+					log_and_message_admins("has modified the payroll of the [department_input] department's members by [num_input].")
+				if("crew_member")
+					var/account_input = input(ui.user, "Account Number", "Please Specify Account Number.") as null|num
+					var/datum/money_account/account = GLOB.station_money_database.find_user_account(account_input, FALSE)
+					if(!account)
+						return
+					accounts_to_modify += account
+			if(!length(accounts_to_modify))
+				return
+			var/raw_input = input(ui.user, "Add message (Max 50 Characters):", "If you wish the owner(s) of the account(s) to recieve a message, please specify") as null|message
```
Comment (farie82) [inline code/modules/economy/economy_admin.dm:72]: Prob want to get the input before you log stuff of it happening

**Comment by Sirryan2002** [comment]
will try and get to this soon, I am entering finals week so I am a bit busy

File: code/modules/economy/economy_admin.dm (hunk 2 above)
Comment (Sirryan2002) [reply code/modules/economy/economy_admin.dm]: fixed in next commit

**Review by farie82 — APPROVED** [review]

**Pull request merged**